packages/dito/app/routes/vorpruefung.einschaetzung.$fileName[.pdf].tsx

```diff
@@ -0,0 +1,132 @@
+import { LoaderFunctionArgs } from "@remix-run/node";
+import * as fs from "node:fs";
+import * as path from "node:path";
+import { PDFDocument, PDFForm, PDFTextField } from "pdf-lib";
+
+const FIELD_NAME_POLICY_TITLE: string = "Titel des Regelungsvorhabens";
```

Comment:
The explicit type annotations here are superfluous if I am not mistaken, typescript will infer them automatically: https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#type-annotations-on-variables

Reply:
True, this was taken from Merlin's code and I didn't question it enough. Good catch 👍